modules/nf-core/cnvkit/export/tests/main.nf.test

```diff
@@ -0,0 +1,191 @@
+
+nextflow_process {
+
+    name "Test Process CNVKIT_EXPORT"
+    script "../main.nf"
+    process "CNVKIT_EXPORT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "cnvkit"
+    tag "cnvkit/export"
+
+    config "./nextflow.config"
+
+    test("test_cnvkit_export_bed") {
+
+        when {
+            params {
+                cnvkit_export_args = "bed -i test"
+            }
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file("https://raw.githubusercontent.com/etal/cnvkit/v0.9.9/test/formats/amplicon.cns", checkIfExists: true)
```

Comment:
Is there no test data in the test data repo? If so, can you add the data there instead?

Reply:
This should be done now for all cnvkit modules.